packages/firebase/index.d.ts

```diff
@@ -1772,6 +1772,15 @@ declare namespace firebase.remoteConfig {
    * Defines levels of Remote Config logging.
    */
   export type LogLevel = 'debug' | 'error' | 'silent';
+  /**
+   * An async function that returns true if current browser context supports initialization of remote config module
```

Comment:
Minor: we know it's an async function from the return type; something like "Returns true if current browser context supports initialization of remote config module (`firebase.remoteConfig()`)" seems sufficient.